README.md

```diff
@@ -7,4 +7,25 @@ Tafkars stands for "The API formerly known as...", is written in Rust and is pro
 
 ## Help wanted
 
-There is still lots of work to be done, especially on [`tafkars-lemmy`](tafkars-lemmy). Pull requests welcome!
\ No newline at end of file
+There is still lots of work to be done, especially on [`tafkars-lemmy`](tafkars-lemmy). Pull requests welcome!
+
+## Quickstart Guide
+
+This Quickstart guide assumes you are using [this fork of libreddit](https://github.com/derivator/libreddit) and [lemmy.world](lemmy.world) to quickly get up and running with tafkars.
```

Comment:
Can we make this sound less like `libreddit` is a requirement? You mention it later, but here it seems required. Something like "`tafkars-lemmy` should work with any Reddit client that allows you to set the API url. In this guide for quick testing on the same machine, we will use this fork of libreddit."